process-migration-service/src/main/resources/META-INF/persistence.xml

```diff
@@ -1,18 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <persistence version="2.1"
-	xmlns="http://xmlns.jcp.org/xml/ns/persistence"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
-	<persistence-unit name="migration-unit"
-		transaction-type="JTA">
-		<provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
-		<class>org.kie.processmigration.model.Migration</class>
-		<class>org.kie.processmigration.model.Plan</class>
-		<class>org.kie.processmigration.model.MigrationReport</class>
-		<properties>
-			<property name="hibernate.max_fetch_depth" value="3" />
-			<property name="hibernate.hbm2ddl.auto" value="create" />
```

Comment:
This part has been omitted. So now Hibernate will not create database tables for PIM schema. I guess we should leave create here for now unless we want to create DDL scripts for each database.

Reply:
If we leave `create` each time the application starts, the database schema and data will be dropped. What do you suggest?